extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftProcessor.java

```diff
@@ -526,9 +526,13 @@ private static RuntimeException openshiftException(Throwable t) {
     }
 
     private static void display(Reader logReader) throws IOException {
```

Comment:
I think this method is only used when there is an IOException and before, we're printing the warning log: 
```java
LOG.warn("Log stream closed, redisplaying last " + LOG_TAIL_SIZE + " entries:");
```

So, I would not add this method and instead will use the Level.WARNING instead:
```
display(client.builds().withName(buildName).tailingLines(LOG_TAIL_SIZE).getLogReader(), Level.WARN);
```

So, it's consistent with the previous trace.

Reply:
I didn't want to introduce "breaking" change. Are you OK with it ? As now it was using `INFO` level in case of IOException (so user can see potential problems).

Reply:
If the INFO traces are printed, then the warning traces should be too, so I don't see a regression here. 
Moreover, I think a warning trace makes more sense here than an INFO trace.

Reply:
I'm not sure if we understand each other.

This [line](https://github.com/quarkusio/quarkus/blob/main/extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftProcessor.java#L484) is using `INFO` and you are suggesting i should change it to `WARN` which will be breaking change, won't be ?

Reply:
why is it a breaking change? Even though it changes to the warning level, it will be printed.

Reply:
Ahh, you are right. I've somehow mixed the precedence. I will change it.